src/clients/BundleDataClient/BundleDataClient.ts

```diff
@@ -1362,14 +1423,26 @@ export class BundleDataClient {
   // keccak256 hash of the relay data, which can be used as input into the on-chain `fillStatuses()` function in the
   // spoke pool contract. However, this internal function is used to uniquely identify a bridging event
   // for speed since its easier to build a string from the event data than to hash it.
-  private getRelayHashFromEvent(event: V3DepositWithBlock | V3FillWithBlock | SlowFillRequestWithBlock): string {
+  protected getRelayHashFromEvent(event: V3DepositWithBlock | V3FillWithBlock | SlowFillRequestWithBlock): string {
```

Comment:
Calls to this will break once `V3FillWithBlock` has a `messageHash` instead of a message. I think you should hash the message if it is a deposit, and then just swap `event.message ` with `event.messageHash` here.

Reply:
Can we do this in a follow up? Otherwise this will break until we upgrade the contracts on-chain

Reply:
https://github.com/across-protocol/sdk/pull/850/files should handle this